registry-manager-rest-server/src/main/resources/static/main.js

```diff
@@ -623,6 +676,24 @@ function showButton() {
     icon.classList.toggle("rotate");
 }
 
+const proceedButton = () => {
+    if (HIT_BUTTON === "UPLOAD") {
+        upload();
+    } 
+    if (HIT_BUTTON === "MERGE") {
+        merge();
+    }
```

Comment:
Maybe it's better to use `if...else...` instead of multiple `if`-s to not to perform unnecessary checks (for instance, if `HIT_BUTTON` is `UPLOAD`, we'll make one more check `if (HIT_BUTTON === "MERGE")`), what do you think?

Reply:
Good point, I'll amend it

Reply:
done